includes/Core/Util/Remote_Features.php

```diff
@@ -83,14 +74,12 @@ final class Remote_Features {
 	 *
 	 * @since n.e.x.t
 	 *
+	 * @param Options        $options        Options instance.
 	 * @param Authentication $authentication Authentication instance.
-	 * @param Context        $context        Plugin context.
-	 * @param Options        $options        Optional. Option API instance. Default is a new instance.
 	 */
-	public function __construct( Authentication $authentication, Context $context, Options $options = null ) {
```

Comment:
Our convention has been to pass "primitive" dependencies before more complex instances with `Context` is always first (as one of the most fundamental). Context also isn't used here other than to instantiate the optional Options instance which probably shouldn't be optional.